deploy.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+ORGANIZATION=rajeshlaerdal
+REPOSITORY_NAME=anonymizertest
```

Comment:
REPOSITORY_NAME should be a parameter

Reply:
Moved these to yaml file for easy configuration